WFInfo/Data.cs

```diff
@@ -1231,7 +1231,7 @@ public async Task GetUserLogin(string email, string password)
                 RequestUri = new Uri("https://api.warframe.market/v1/auth/signin"),
                 Method = HttpMethod.Post,
             };
-            var content = $"{{ \"email\":\"{email}\",\"password\":\"{password.Replace(@"\", @"\\")}\", \"auth_type\": \"header\"}}";
+            var content = $"{{ \"email\":\"{email}\",\"password\":\"{password.Replace(@"\", @"\\").Replace("\"", "\\\"")}\", \"auth_type\": \"header\"}}";
```

Comment:
while we're at it, perhaps it's time to construct json via json libraries instead of doing it with formatting/concatenation.